services/scheduler/scheduler.go

```diff
@@ -95,7 +102,7 @@ func (s *Service) Add(task Task, params AddParams) (*models.ScheduledTask, error
 			return err
 		}
 
-		if err := s.addDBTask(scheduledTask); err != nil {
+		if err = s.addDBTask(scheduledTask); err != nil {
```

Comment:
addDBTask call shouldn't be in the transaction since it doesn't accept tx.

Reply:
`addDBTask` doesn't use DB at all, it changes the in-memory state.  There is rollback functionality for this state here :https://github.com/percona/pmm-managed/blob/9a5d465d7cdccc48f1403ef916babce209deb1a6/services/scheduler/scheduler.go#L112-L118 but most likely it will not work properly. 

Anyway, all this stuff is not part of the current ticket, so it should be refactored in different ticket/pr